### PR TITLE
Added Banner, Social Links, Info, Contact

### PR DESCRIPTION
**Pull request by Ayush7614**
This PR Contains

* Added Banner
* Social Links
* Contact Links
* Added Projects

### CODE REVIEW

**Comment by Ayush7614** [comment]
This PR is for the Issues #1

File: profile/README.md
Comment (joewxboy) [inline profile/README.md:19]: Instead of "Here are few things which we have developed in last couple of years." let's say:
"Here are some steps to learn and use the Open Horizon component services."
1. [Install the Management Hub Services](https://github.com/open-horizon/devops/blob/master/mgmt-hub/README.md#-deploy-all-in-1-horizon-management-hub-agent-and-cli) on an x86 machine or VM using our easy all-in-one script.
2. [Install the Agent](https://github.com/open-horizon/devops/blob/master/mgmt-hub/README.md#adding-more-edge-devices) on almost any single-board computer (SBC), desktop, laptop, or Kubernetes cluster.
3. Publish and use [our example services](https://github.com/open-horizon/examples/tree/master/edge/services).
4. [Create and publish your own services](https://github.com/open-horizon-services/Getting-Started#contributing-a-new-microservice).

File: profile/README.md
Comment (joewxboy) [inline profile/README.md:27]: Instead of "on the respective projects", how about "in the respective repositories"

and let's link maintainers, contributors and other community members to the community page on the wiki that explains what those roles are: https://wiki.lfedge.org/display/OH/Community+Membership

File: profile/README.md
Comment (joewxboy) [inline profile/README.md:38]: Let's add a statement at the bottom:

Open Horizon is a Stage Two: Growth project at [LF Edge](https://www.lfedge.org), an umbrella group within the Linux Foundation.

**Review by joewxboy — CHANGES_REQUESTED** [review]
I've added some comments about proposed changes inline.

**Comment by joewxboy** [comment]
@Ayush7614 Your latest commit may have been missing a sign-off.  Can you amend that commit to add it?  GitHub won't let me review your changes.

**Comment by joewxboy** [comment]
@Ayush7614 See this message: https://github.com/open-horizon/.github/pull/2/checks

**Comment by Ayush7614** [comment]
> @Ayush7614 See this message: https://github.com/open-horizon/.github/pull/2/checks

Give me some minute i am doing all the things

**Comment by Ayush7614** [comment]
> > @Ayush7614 See this message: https://github.com/open-horizon/.github/pull/2/checks
> 
> Give me some minute i am test all the things

cc @joewxboy Please Review this and merge it if all things are correct

**Review by joewxboy — APPROVED** [review]

**Pull request merged**
